README.rst

```diff
@@ -140,16 +145,25 @@ maintain which implementation ``cwl-runner`` points to via a symbolic file
 system link or `another facility <https://wiki.debian.org/DebianAlternatives>`_.
 
 Recommended Software
-^^^^^^^^^^^^^^^^^^^^
+====================
+
+You strongly suggested to have the following installed:
+
+* One of the following software container engines
 
-You may also want to have the following installed:
-- `node.js <https://nodejs.org/en/download/>`_
-- Docker, udocker, or Singularity (optional)
+  * Podman
+  * `Docker <https://docs.docker.com/engine/install/>`_
+  * Singularity/Apptainer: See `Using Singularity`_
+  * udocker: See `Using uDocker`_
+
+* `node.js <https://nodejs.org/en/download/>`_ for evaluating CWL Expressions quickly
+  (required for `udocker` users, optional but recommended for other container engines (podman/d
```

Comment:
I think the end, (podman/d is wrong? maybe other container engines e.g. podman) ?